src/test/java/org/broadinstitute/hellbender/utils/locusiterator/LocusIteratorByStateBaseTest.java

```diff
@@ -17,12 +24,41 @@
     protected static SAMFileHeader header;
     protected GenomeLocParser genomeLocParser;
 
+    /**
+     * For testing only.  Assumes that the incoming SAMRecords have no read groups, so creates a dummy sample list
+     * for the system.
+     */
+    public static List<String> sampleListForSAMWithoutReadGroups() {
+        List<String> samples = new ArrayList<>();
+        samples.add(null);
+        return samples;
+    }
+
     @BeforeClass
     public void beforeClass() {
         header = ArtificialReadUtils.createArtificialSamHeader(1, 1, 1000);
         genomeLocParser = new GenomeLocParser(header.getSequenceDictionary());
     }
 
+    protected LocusIteratorByState makeLIBS(final List<GATKRead> reads, final SAMFileHeader header) {
+        return makeLIBS(reads, null, false, header);
+    }
+
+    protected LocusIteratorByState makeLIBS(final List<GATKRead> reads,
+                                            final DownsamplingMethod downsamplingMethod,
+                                            final boolean keepUniqueReadList,
+                                            final SAMFileHeader header) {
+        reads.sort(new ReadCoordinateComparator(header));
```

Comment:
This comparator won't interleave unmapped reads that have the position of their mates with mapped reads, as they would be returned from a real BAM file. Does this have implications for the LIBS tests, or should we not worry since LIBS doesn't operate on unmapped reads?